gengokit/httptransport/types.go

```diff
@@ -49,12 +50,14 @@ type Field struct {
 	// The location within the the http request that this field is to be found.
 	Location string
 	// The protobuf type that this field is of.
+	// Deprecated by Svcdef.
 	ProtobufType string
 	// The type within the Go language that's used to represent the original
 	// field that this field refers to.
 	GoType string
 	// The protobuf label for the original field that this field refers to. Is
 	// probably "OPTIONAL", though may be "REPEATED".
+	// Deprecated by Svcdef.
```

Comment:
Ha, at the time I wrote that for my own benefit with the intention of erasing those field members entirely once I'd moved on. So I'm going to just remove those, as they're not used anywhere.